amethyst_rendy/src/types.rs

```diff
@@ -14,8 +14,8 @@ macro_rules! impl_backends {
 
         impl_single_default!($([$feature, $backend]),*);
 
-        #[cfg(not(any($(feature = $feature),*)))]
-        pub type DefaultBackend = rendy::empty::Backend;
+        //#[cfg(not(any($(feature = $feature),*)))]
+        //pub type DefaultBackend = rendy::empty::Backend;
```

Comment:
just remove those comments

Reply:
Or actually, we should have the static_assert here instead. Something like that
```rs
 static_assertions::assert_cfg!( 
     any($(feature = $feature),*),
     concat!("You must specify a at least one graphical backend feature:", stringify!($( '$feature'),*))
)

Reply:
Applied in latest commit.